sesman/session.c

```diff
@@ -37,10 +40,22 @@
 #include <sys/prctl.h>
 #endif
 
+#include <sys/mman.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <pthread.h>
```

Comment:
Are all those includes really needed? we better handle pthread stuff inside thread_calls.c